cvat-ui/src/components/actions-menu/dump-submenu.tsx

```diff
@@ -19,15 +19,17 @@ interface Props {
     menuKey: string;
     dumpers: any[];
     dumpActivities: string[] | null;
+    taskDimension: string;
 }
 
 export default function DumpSubmenu(props: Props): JSX.Element {
-    const { taskMode, menuKey, dumpers, dumpActivities } = props;
+    const { taskMode, menuKey, dumpers, dumpActivities, taskDimension } = props;
 
     return (
         <Menu.SubMenu key={menuKey} title='Dump annotations'>
             {dumpers
                 .sort((a: any, b: any) => a.name.localeCompare(b.name))
+                .filter((dumper: any): JSX.Element => dumper.dimension === taskDimension)
```

Comment:
filter callback returns boolean, not JSX.Element

Reply:
Made changes accordingly.